contracts/BSCValidatorSet.sol

```diff
@@ -95,6 +95,10 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     Validator[] private _tmpMigratedValidatorSet;
     bytes[] private _tmpMigratedVoteAddrs;
 
+    // BEP-341 Validators can produce consecutive blocks
+    uint256 public turnTerm;
```

Comment:
Can we write an exact definition of `turnTerm` in comment here?

Reply:
> Can we write an exact definition of `turnTerm` in comment here?

accepted